x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -30,13 +30,12 @@ const formatTooltipValue = (coordinate: Coordinate) => {
     : NOT_AVAILABLE_LABEL;
 };
 
-const trackHoverBreakdownChart = throttle(
-  () => trackEvent({ app: 'apm', name: 'hover_breakdown_chart' }),
-  60000
-);
-
 const TransactionBreakdownGraph: React.FC<Props> = props => {
   const { timeseries } = props;
+  const trackApmEvent = useUiTracker({ app: 'apm' });
+  const trackHoverBreakdownChart = throttle(() => {
```

Comment:
Doesn't this create a newly wrapped function (and thereby resetting the throttling timer) on every rerender? https://github.com/streamich/react-use/blob/master/docs/useThrottle.md could be used here to avoid that.

Reply:
Yeah, good catch.